symbol-processing/build.gradle.kts

```diff
@@ -68,5 +68,15 @@ publishing {
                 }
             }
         }
+
+        create<MavenPublication>("cmdline") {
```

Comment:
should this name be `native` instead?

Reply:
No, because it's not native specific; It's more like *unrelocated*. 

FWIW, I was considering renaming:
`symbol-processing` -> `symbol-processing-embeddable`
`symbol-processing-cmdline` -> `symbol-processing`

Because they shouldn't be used directly by users. Well, maybe with the exception of `kotlin-compile-testing`.

Naming is too hard. So finally I settled down with `symbol-processing-cmdline`. Any thoughts?

Reply:
I see, I was having that impression due to the description. Feel free to merge.